ChangeLog.rst

```diff
@@ -27,13 +27,13 @@ Configuration from the CLI is:
 - `az account set --subscriptions <https://docs.microsoft.com/en-us/cli/azure/manage-azure-subscriptions-azure-cli>`__: access to subscription_id
 - `az cloud set --name <https://docs.microsoft.com/en-us/cli/azure/cloud#set>`__: access to base_url (sovereign cloud, Government, Germany, etc.)
 
-Note: CLI and SDK have different versionning story. We do recommend using two different virtual environment. Information will be shared
-automatically since the CLI saves them in the $HOME folder. You do require to install the `azure-cli-core` package in your SDK environment to use this feature.
+Note: CLI and SDK have different versioning story. We do recommend using two different virtual environments. Information will be shared
```

Comment:
"Story" is a little ambiguous: how about "The CLI and SDK are versioned separately."